README.md

```diff
@@ -89,7 +92,7 @@ dataset = nums.read_csv("path/to/csv", has_header=True)
 ```
 
 ##  Logistic Regression
-
+(Try this example live in your [browser](https://mybinder.org/v2/gh/nums-project/nums-binder-env/master?urlpath=git-pull%3Frepo%3Dhttps%253A%252F%252Fgithub.com%252Fnums-project%252Fnums%26urlpath%3Dtree%252Fnums%252Fexamples%252Fnotebooks%252Flogistic_regression.ipynb%26branch%3Dmaster)) \
```

Comment:
- Currently, this link doesn't work. It looks like it will work properly once the PR is merged.
- Could we have this launch in a new browser window?
- Could we make this link a part of a single sentence at the end of this section? Something like "You can run this example in your browser **here**."

Reply:
> * Currently, this link doesn't work. It looks like it will work properly once the PR is merged.
Yes
> * Could we have this launch in a new browser window?
i have looked into this issue in the past too. Markdown does not support this so far
> * Could we make this link a part of a single sentence at the end of this section? Something like "You can run this example in your browser **here**."
Yes. Pushed the changes.

Reply:
> * Currently, this link doesn't work. It looks like it will work properly once the PR is merged.

yes


> * Could we have this launch in a new browser window?

i have looked into this issue in the past too. Markdown does not support this so far


> * Could we make this link a part of a single sentence at the end of this section? Something like "You can run this example in your browser **here**."

done

Reply:
dont merge this just yet. i rebased and lost some changes. fixing them